kaztron/cog/dice.py

```diff
@@ -84,6 +86,25 @@ def __init__(self, bot):
         await self.bot.say("[{}]\n**Sum:** {:d}".format(' '.join(rolls_str), total))
         logger.info("Rolled FATE dice: {!r} (sum={})".format(rolls_str, total))
 
+    # strip_list = map(str.strip, lines)
+    @commands.command(pass_context=True, ignore_extra=False, no_pm=False)
+    @in_channels(ch_allowed_list)
```

Comment:
IMO since .choose isn't a dice roll, for worldbuilding's purposes we could allow this in all channels - IMO remove this line (see also the PM comment above)